src/components/SchedulePage/Schedule.js

```diff
@@ -6,17 +6,21 @@ import List from '@material-ui/core/List';
 import Divider from '@material-ui/core/Divider';
 import StickyTimeSlot from '../SchedulePage/StickyTimeSlot';
 import workshopSchedule from '../../data/WorkshopSchedule';
-import ComingSoon from '../ComingSoon/ComingSoon';
+// import ComingSoon from '../ComingSoon/ComingSoon';
 
 // When changing this, make sure to update src/components/HomePage/UpcomingEvents.js as well.
-const scheduleAvailable = false;
+const scheduleAvailable = true;
 
 const useStyles = makeStyles(theme => ({
 	title: {
 		fontWeight: 'bold',
 		paddingTop: theme.spacing(11),
 		paddingBottom: theme.spacing(scheduleAvailable ? 4 : 11)
 	},
+	note: {
+		color: '#858585',
```

Comment:
See my comment about the Typography variant below.

Not sure what the different variants render as, but the color might change depending on the variant you use so be aware of that when you experiment.